DEV-2648: Adding proposal for Glen Dimplex parameters. (#124)
futurehomeno/fimp-api: device_services/meta/parameters.md

| `cable_always_locked`     | Cable always locked     | `bool`     | `chargepoint` service                     | Allows to enforce charging cable to be always locked on the EVSE side to prevent theft.                                                                                                                                                          |
| `free_charging`           | Free charging           | `bool`     | `chargepoint` service                     | Determines whether chargepoint should automatically start charging an EV without an authorization or remote control trigger.                                                                                                                     |
| `led_brightness`          | LED brightness          | `int`      | `chargepoint` service                     | Allows to set the brightness of the LED ring on the chargepoint. Accepted values are defined by `min` and `max` properties of the parameter definition.                                                                                          |

> | `device_lock`             | Device lock             | `int`      | `thermostat` or `power_regulator` service | Configure lock setting of the heating device.                                                                                                                                                                                                    |

Reviewer: ```suggestion
| `device_lock`             | Device lock             | `int`      | `thermostat` or `power_regulator` service | Configure lock setting of the thermostat device.                                                                                                                                                                                                    |
```

Author: Done

> | `used_sensors`            | Used sensor             | `int`      | `thermostat` service                      | Set sensor type,which will be use to calculate heater setting                                                                                                                                                                                    |

Reviewer: ```suggestion
| `used_sensors`            | Used sensor             | `int`      | `thermostat` service                      | Select which sensor to use. This will be use to calculate heater setting.                                                                                                                                                                                    |
```

> | `floor_sensor_resistance` | Floor sensor resistance | `int`      | `thermostat` service                      | Set he kΩ floor sensor rating, if is installed                                                                                                                                                                                                   |

Reviewer: ```suggestion
| `floor_sensor_resistance` | Floor sensor resistance | `int`      | `thermostat` service                      | Set the kΩ floor sensor rating, if is installed.                                                                                                                                                                                                   |
```

Author: Done
